boards/arm/ubx_bmd340eval_nrf52840/Kconfig

```diff
@@ -0,0 +1,10 @@
+# BMD -340-EVAL board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX -License-Identifier: Apache-2.0
```

Comment:
There's an extra space between "SPDX" and "-License-Identifier" that needs to be removed

Reply:
Fixed, also removed space in BMD-340-EVAL in line 1

Reply:
It's fixed, but in the wrong commit. :( It's currently in `board: arm: updating bmd340 submission for adc tests` but should be in `board: arm: add ubx_bmd340eval_nrf52840`